Cargo.toml

```diff
@@ -8,26 +8,26 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-csv = "1.1.3"
+csv = "1.2"
 # we need new version to enable static builds
 fasthash = "0.4"
-serde = {version = "1.0.114" , features = ["derive"]}
-serde_json = "1.0.55"
+serde = {version = "1.0.163" , features = ["derive"]}
+serde_json = "1.0.96"
 clap = "2.33.1"
-byteorder = "1.3.4"
+byteorder = "1.4.3"
 merand48 = "0.1.0"
-daemonize = "0.4.1"
-lz4 = "1.23.2"
-nom = "7"
-dyn-clone = "1.0"
+daemonize = "0.5.0"
+lz4 = "1.24.0"
+nom = "7.1.3"
+dyn-clone = "1.0.11"
 rand = "0.8.5"
 rand_distr = "0.4.3"
 rand_xoshiro = "0.6.0"
-flate2 = { version = "1.0", features = ["cloudflare_zlib"], default-features = false }
+flate2 = { version = "1.0.26", features = ["zlib-ng"], default-features = false }
```

Comment:
I think this change will specifically affect the speed of the cache usage... So make sure your benchmarks include using the cache

Reply:
Good point. Re-tested this part once more, it's actually a bit (few percent) faster consistently still